packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue

```diff
@@ -292,18 +298,19 @@
       },
 
       cancelBtn() {
-        this.$router.push({ name: PageNames.ManageSyncSchedule });
+        this.$router.push({ name: SyncPageNames.MANAGE_SYNC_SCHEDULE });
```

Comment:
This will lose the context of the facility in case the route needs one - we may be better off passing in this 'back' and 'cancel' routes as parameters that we can configure in the different plugins.